### ISSUE TITLE
Inconsistent support for mailmap.file configuration

### ISSUE THREAD

**Issue by AFulgens** [ForkIssues/TrackerWin#2274]
### Context

Git offers the configuration entry `mailmap.file`, which defines a path to a [mailmap file](https://git-scm.com/docs/gitmailmap), which may be outside of the repository.

Currently, Fork (on Windows at least) respects this setting at multiple places already:

- commit panel (default view bottom right panel)
- repository statistics

However, it does not support it everywhere, for example in the main panel (default view top right panel) it is not supported. However, the panel supports mailmap, in case a file `.mailmap` is in the git repository itself.

### Feature request

The config entry `mailmap.file` should be supported consistently everywhere, but at least the main panel should be added to the list of widget that respects it.

**Comment by DanPristupov** [comment]
> Git offers the configuration entry `mailmap.file`

Currently mailmap defined in git config `mailmap.file` is not supported (https://github.com/fork-dev/TrackerWin/issues/2182#issuecomment-2024629311). Only the `.mailmap` configuration in the repository root is supported.

**Comment by AFulgens** [comment]
Not supported in the sense of... "it just happens to work" for the two things I mentioned in the issue opening comment? Because it works, I see it working :)

**Comment by DanPristupov** [comment]
Yes, I know, it works somewhere, but the commit list is different. There is a non-trivial internal problem which doesn't allow us to fix it right now.

> Not supported in the sense of... "it just happens to work" for the two things I mentioned in the issue opening comment? 

Well, yes, you are right :).

**Comment by AFulgens** [comment]
Thanks for taking the time to answer!

Sad to read this, but it is what it is.

Nota bene: IntelliJ IDEA is somewhat a bigger project than Fork and even they don't have support for it. Better said: They support it in inline annotations, but not in the main Git overview panel, even if it is a `.mailmap` file in the root of the repo 😁 And their tickets are like 10+ years old. Same with GitLab, their support is also sporadic currently. So... what I maybe want to say: Keep up the good work and I wish you a very happy weekend.

**Comment by DanPristupov** [comment]
We will fix this one day, but I don't have an ETA because currently we are working on more important things.

Thank you for the kind words :).
